frontend/src/config/i18n/en/drawings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "tools_title": "Choose Shape",
+  "draw_rectangle_hint": "Use mouse to draw a rectangle.",
+  "draw_ellipse_hint": "Use mouse to draw an ellipse.",
+  "draw_free_line_hint": "Use mouse to draw a line.",
+  "draw_bezier_polygon_hint": "Use left mouse button to add points, right mouse button to finish current polygon.",
```

Comment:
Consider using the term 'left-click' and 'right-click' instead of 'left mouse button' or 'right mouse button'.
This would make the messages more concise and I personally think it sounds better.